generators/express/templates/circleci.yaml.ejs

```diff
@@ -23,6 +23,17 @@ jobs:
       - run:
           command: yarn lint
 
+  <%= packageName %>-test:
+    executor: node
+    working_directory: ~/project/<%= packagePath %>
+    steps:
+      - checkout:
+          path: ~/project
+      - attach_workspace:
+          at: ~/project
+      - run:
+          command: yarn test
```

Comment:
Is it possible to display information about test coverage in circelci ? 
If so, It may be better to run command `yarn test:coverage`

Reply:
We can export them as artifacts but CircleCI doesn't have an interface to display them properly (and particularly display the variation in coverage).